contracts/meta-oracles/lib/TimeSeriesStateLibrary.sol

```diff
@@ -0,0 +1,33 @@
+/*
+    Copyright 2019 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+pragma experimental "ABIEncoderV2";
+
+
+/**
+ * @title TimeSeriesStateLibrary
+ * @author Set Protocol
+ *
+ * Library defining TimeSeries state struct
+ */
+library TimeSeriesStateLibrary {
+    struct State {
+        uint256 nextEarliestUpdate;
+        uint256 updateInterval;
+        uint256[] previousLoggedPrices;
```

Comment:
Why `previousLoggedPrices` vs `timeSeriesData`?

Reply:
Feels like both are equally descriptive. Why would you choose one over the other?

Reply:
Also `timeSeriesData` is the variable name of the `LinkedList` on `TimeSeriesFeed`